src/views/homepage/components/page-title/page-title.module.css

```diff
@@ -50,6 +50,5 @@
 	color: var(--token-color-foreground-high-contrast);
 	display: block;
 	font-weight: var(--token-typography-font-weight-semibold);
-	margin-top: 10px;
```

Comment:
🗒️ Just removed space between the two pieces of the page title, at the request of Design.